service/controller/v3/resource/instance/template/main.json

```diff
@@ -0,0 +1,258 @@
+{
+  "$schema":"https://schema.management.azure.com/schemas/2015-01-01/deploymentTemplate.json#",
+  "contentVersion":"1.0.0.0",
```

Comment:
Here are the separated templates. I ran a formatter against them so this is now consistent. I would like to do this with the other templates in a separate PR as well, because right now it is quite a mess.